### PR TITLE
Interface added

### PR DESCRIPTION
**Pull request by TomWayne98**

### CODE REVIEW

File: library/src/main/java/com/github/clans/fab/ExtendedFloatingActionMenu.java [hunk 1]
```diff
@@ -998,6 +998,14 @@ public void setMenuButtonColorPressedResId(int colorResId) {
         mMenuButton.setColorPressedResId(colorResId);
     }
 
+    public float getMenuX() {
```
Comment (davidvavra) [inline library/src/main/java/com/github/clans/fab/ExtendedFloatingActionMenu.java:1002]: Missing override

File: library/src/main/java/com/github/clans/fab/ExtendedFloatingActionMenu.java [hunk 2]
```diff
@@ -998,6 +998,14 @@ public void setMenuButtonColorPressedResId(int colorResId) {
...
+        return getPivotX();
+    }
+
+    public float getMenuY() {
```
Comment (davidvavra) [inline library/src/main/java/com/github/clans/fab/ExtendedFloatingActionMenu.java:1007]: Missing override

File: library/src/main/java/com/github/clans/fab/FloatingActionMenu.java [hunk 3]
```diff
@@ -92,7 +92,7 @@
     private boolean mIsSetClosedOnTouchOutside;
     private boolean animationInProgress = false;
     private int mOpenDirection;
-    private OnMenuToggleListener mToggleListener;
+    private ExtendedFloatingActionMenu.OnMenuToggleListener mToggleListener;
```
Comment (davidvavra) [inline library/src/main/java/com/github/clans/fab/FloatingActionMenu.java]: Move this from ExtendedFloatingActionMenu class and rename it to FloatingMenuToggleListener

File: library/src/main/java/com/github/clans/fab/FloatingActionMenu.java [hunk 4]
```diff
@@ -715,6 +716,14 @@ public void run() {
         }
     }
 
+    public float getMenuX() {
```
Comment (davidvavra) [inline library/src/main/java/com/github/clans/fab/FloatingActionMenu.java:720]: Missing override

File: library/src/main/java/com/github/clans/fab/FloatingActionMenu.java [hunk 5]
```diff
@@ -715,6 +716,14 @@ public void run() {
...
+        return getPivotX();
+    }
+
+    public float getMenuY() {
```
Comment (davidvavra) [inline library/src/main/java/com/github/clans/fab/FloatingActionMenu.java:725]: Missing override

File: library/src/main/java/com/github/clans/fab/FloatingMenu.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
+package com.github.clans.fab;
+
+import android.animation.AnimatorSet;
+import android.view.View;
+import android.view.animation.Animation;
+import android.widget.ImageView;
+
+/**
+ * WRITE DESCRIPTION PLS
```
Comment (davidvavra) [inline library/src/main/java/com/github/clans/fab/FloatingMenu.java]: Missing description. (if you have Android studio configured that it generates this text, add TODO in the template. This way you can easily check that you haven't forgotten it)

**Review by davidvavra — CHANGES_REQUESTED** [review]

File: library/src/main/java/com/github/clans/fab/ExtendedFloatingActionMenu.java (hunk 1 above)
Comment (TomWayne98) [reply library/src/main/java/com/github/clans/fab/ExtendedFloatingActionMenu.java:1002]: I dont get it. It is used. These methods use interface.

File: library/src/main/java/com/github/clans/fab/ExtendedButtonShadowShape.java [hunk 7]
```diff
@@ -16,7 +16,7 @@
 public class ExtendedButtonShadowShape extends RectShape {
     private Context mContext;
     private float mButtonPadding;
-    private float mExtraLeftShadow = -5f;
+    private float mExtraLeftShadow = -3f;
     private float mExtraTopShadow = -0f;
     private float mRadius = 70f;
```
Comment (TomWayne98) [inline library/src/main/java/com/github/clans/fab/ExtendedButtonShadowShape.java:21]: It caused that few pixel on  left edge of button were missing. It happend only on devices with very low pixel density. This change doesn't cause any problem on other devices.

**Pull request merged**